message_ix/model/MESSAGE/sets_maps_def.gms

```diff
@@ -249,8 +247,7 @@ Sets
     type_emission                           types of emission aggregations
     cat_emission(type_emission,emission)    mapping of emissions to respective categories
     type_tec_land(type_tec)                 dynamic set whether emissions from land use are included in type_tec
-    commodity_surplus(commodity)            subset of 'commodity' to mark all commodities in Equation COMMODITY_BALANCE_GT
-    commodity_shortage(commodity)           subset of 'commodity' to mark all commodities in Equation COMMODITY_BALANCE_LT
+    balance_equality(commodity,level)       subset of 'commodity' and 'level' to mark all commodities in Equation COMMODITY_BALANCE_LT
```

Comment:
rather than write "to mark all commodities in Equation COMMODITY_BALANCE_LT", why not write "mapping  of commodities-level where the supply-demand balance must be maintained with equality"